R/eDISH.R

```diff
@@ -47,6 +49,20 @@
 #'                             AST = "Aspartate Aminotransferase (U/L)",
 #'                             TB = "Bilirubin (umol/L)",
 #'                             ALP = "Alkaline Phosphatase (U/L)"))
+#' 
+#' ## Create eDISH figure using a premade settings list
```

Comment:
The following example may be useful to illustrate using group_cols to color the points and adding the filters. It matches the safety-edish built-in example

```r
# Add group_cols & filters to example
group_cols_vec <- c("TRTP","SEX", "AGEGR1")

filters_df <- data.frame(
  value_col=c("TRTA", "SEX", "RACE", "AGEGR1"),
  label = c("Treatment", "Sex", "RACE", "Age group")
)


settingsl <- list(id_col = "USUBJID",
                  value_col = "AVAL", 
                  measure_col = "PARAM", 
                  group_cols = group_cols_vec,
                  filters = filters_df, 
                  visitn_col = "VISITNUM", 
                  normal_col_low = "A1LO", 
                  normal_col_high = "A1HI", 
                  measure_values = list(ALT = "Alanine Aminotransferase (U/L)",
                  AST = "Aspartate Aminotransferase (U/L)",
                  TB = "Bilirubin (umol/L)",
                  ALP = "Alkaline Phosphatase (U/L)"))

eDISH(data=adlbc, settings = settingsl)
```